src/sentry/event_manager.py

```diff
@@ -521,6 +532,27 @@ def save(self, project_id, raw=False, assume_normalized=False, start_time=None,
         return job["event"]
 
 
+@metrics.wraps("event_manager.background_grouping")
```

Comment:
I think you can inline this function

Reply:
You're right, but I'm mocking it in a unit test so I'd rather leave it as a function.